Resources/Prototypes/Catalog/Cargo/cargo_livestock.yml

```diff
@@ -197,3 +197,13 @@
   cost: 2800
   category: Livestock
   group: market
+
+- type: cargoProduct
+  id: LivestockMothroach
+  icon: mothroach.rsi
+    sprite: Mobs/Animals/mothroach.rsi
+    state: mothroach
+  product: CrateNPCMothroach
+  cost: 2500
```

Comment:
reduce the value of the moths. They're cute, but not so cute that they cost so much money.)

Reply:
the mothconomy will never recover

Reply:
3500 seems more fair, tbh

Reply:
i have to do it anyway but youll get 6 moths per crate now

Reply:
reverted